### ISSUE TITLE
Add command/keyboard shortcut for --export-secret-subkeys

### ISSUE THREAD

**Issue by wararjey** [orhun/gpg-tui#15]
Part of my workflow is having 2 keyrings:
- main keyring with all subkeys, used for signing other keys, only on a single machine.
- keyring with primary secret subkey keys stripped, used for everyday things like en-/decrypting, signing mails etc.

All my own keys do have a 2nd subkey used for signing mails. A few keys of the 2nd keyring do end up being present more than one of my devices...

gpg supports this workflow with the --export-secret-subkeys command, exporting keys with secret part of the primary key stripped. It would be nice if that command would be added to gpg-tui, as well.

Regards,

**Comment by orhun** [comment]
> gpg supports this workflow with the --export-secret-subkeys command, exporting keys with secret part of the primary key stripped. It would be nice if that command would be added to gpg-tui, as well.

So as far as I understand, running `gpg --export-secret-subkeys` for the selected key is enough. There's no need for subkey selection etc. (Correct me if I'm wrong)

I think I can add a menu entry for this purpose.

**Comment by wararjey** [comment]
Yes, that is correct. All subkeys are exported, except for the secret part of the primary key.

However, an additional part of that workflow is that one has to export and import the ownertrust, as well. That is not a big deal, though, as ownertrust can be set by editing the imported key in gpg-tui itself.

**Comment by orhun** [comment]
I added a new menu entry for running `gpg --export-secret-subkeys` on the selected key:

![](https://user-images.githubusercontent.com/24392180/121078214-53a7b180-c7e1-11eb-9878-11f2ec5bc1a6.gif)

Also the new syntax for the `:export` command is: `:export <key_type> <query> (subkey)`

e.g. `:export sec 0x0123 subkey` will export the secret subkeys of `0x0123`. It will be much easier to use these commands when I make TUI more customizable. (#5, #6, etc.)

I created #20 and push a new release soon. We can discuss further about improvements here. Feel free to report if anything's missing/not working.
